docker-compose.yml

```
@@ -57,6 +59,45 @@ services:
       - ./jsonnetfile.json:/go/jsonnetfile.json
       - ./jsonnetfile.lock.json:/go/jsonnetfile.lock.json
     entrypoint: jb
+  
+  ralph:
+    image: fundocker/ralph:latest
+    user: "${DOCKER_USER:-1000}"
```

To ease the service usage, I would recommend:

```suggestion
    user: "${DOCKER_USER:-1000}"
    entrypoint: ["ralph"]
```